recipes/srst2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: srst2
+  version: 0.2.0
+
+source:
+  fn: v0.2.0.tar.gz
+  url: https://github.com/katholt/srst2/archive/v0.2.0.tar.gz
+  sha256: 352c59bdf00039dfa5610a4b88049a0eb114d7c2a305d4924b6e605cc6d15123
+  patches:
+    - use_setuptools.patch
+
+build:
+  number: 0
+  skip: True #[not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+
+  run:
+    - python
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+    - bowtie2 >=2.1.0
+    - samtools ==0.1.18
```

Comment:
setuptools is just needed to be able to install the python package, from looking at other recipes it only appears to be included in the build section.